File: 10708.c
#include <stdio.h>

int main() {
	int N, M;
	int target[101];
	int vote[101];
	int point[101];

	scanf("%d\n", &N);
	scanf("%d\n", &M);
	
	for (int i = 1; i <= M; i++)
		scanf("%d", &target[i]);

	for (int i = 1; i <= N; i++)
		point[i] = 0;

	for (int i = 1; i <= M; i++) {
		for (int j = 1; j <= N; j++) {

			scanf("%d", &vote[j]);
			if (target[i] == vote[j])
				point[j]++;
			else
				point[target[i]]++;
		}
	}

	for (int i = 1; i <= N; i++) {
		printf("%d\n", point[i]);
	}
}

File: 10708.py
n = int(input())
m = int(input())
point = [0 for i in range(0,n)]
#print(point)
target = [int(i)-1 for i in input().split()]
#print(a)
for i_target in range(0,m):
    vote = [int(i)-1 for i in input().split()]
    #print(vote)
    for v_point in range(0,n):
        if target[i_target] == vote[v_point]:
            point[v_point]+=1
        else:
            point[target[i_target]] += 1
    #print(point)
for pt in point:
    print(pt)
